app/src/main/java/com/czech/chronos/ui/components/SavedLocationsList.kt

```diff
@@ -0,0 +1,119 @@
+package com.czech.chronos.ui.components
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.layoutId
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.czech.chronos.network.models.CurrentTime
+import com.czech.chronos.utils.DateUtil
+import com.czech.chronos.utils.Fonts
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.withContext
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun SavedLocationsList(
+	list: List<CurrentTime>,
+	modifier: Modifier = Modifier
+) {
+	LazyColumn(
+		modifier = modifier
+			.layoutId("locations")
+	) {
+		items(
+			items = list
+		) { data ->
+			SavedLocationsItem(
+				data = data,
+				modifier = modifier
+			)
+		}
+	}
+}
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun SavedLocationsItem(
+	data: CurrentTime,
+	modifier: Modifier,
+) {
+
+	val timeFormatter = remember { DateUtil.timeFormat }
+
+	var locationTime by remember {
+		mutableStateOf(
+			ZonedDateTime.now(ZoneId.of(data.timezoneLocation))
+		)
+	}
+
+	LaunchedEffect(Unit) {
```

Comment:
Your code doesn't look formatted. CMD + Shift + L in Android Studio will format it using the official kotlin codestyle

Reply:
Alright